lib/std-gen-seed/std-gen-seed.cabal

```
@@ -0,0 +1,64 @@
+cabal-version:   3.0
+name:            std-gen-seed
+version:         0.1.0.0
```

If we're going to start at the beginning, then we can start at `0.0.0.0`:

```suggestion
version:         0.0.0.0
```